expertise/models/specter2_scincl/predictor.py

```diff
@@ -1,4 +1,7 @@
 from tqdm import tqdm
+import openreview
```

Comment:
why do you need to include openreview? just for triggering the exception? maybe this can be a regular exception